app/src/main/java/com/example/travelwith/travelwithandroidapp/object/AESCrypt.java

```diff
@@ -0,0 +1,38 @@
+package com.example.travelwith.travelwithandroidapp.object;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.util.Base64;
+
+import com.example.travelwith.travelwithandroidapp.connection.Util;
+
+import java.security.Key;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+
+@SuppressLint("GetInstance")
+public class AESCrypt {
+    private static final String ALGORITHM = "AES";
+
+    public static String encrypt(Context context, String value) throws Exception {
+        Key key = new SecretKeySpec(Util.getProperty("AESkey", context).getBytes(), AESCrypt.ALGORITHM);
+        Cipher cipher = Cipher.getInstance(AESCrypt.ALGORITHM);
+        cipher.init(Cipher.ENCRYPT_MODE, key);
+        byte[] encryptedByteValue = cipher.doFinal(value.getBytes("utf-8"));
```

Comment:
Ja bym tego nei encodował w Base64 niech lata zaencryptowane i tyle xD w sumie mi to bez różnicy pytanie jak sie bedziesz logował - z basem czy bez. bo na razie mam to tak zrobione na serwerze ze zapisuje do bazy co mi przyślesz

Reply:
Wow czyli co tablice bytów mam Ci wypchnąć ?